src/main/java/org/CharacterCreator/DataModel/FileWriter.java

```diff
@@ -0,0 +1,50 @@
+package org.CharacterCreator.DataModel;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriter {
+     public String writeFile(String name, org.CharacterCreator.DataModel.Character character, String selection, String background) {
```

Comment:
Race and class names can be passed on from the main method

Reply:
org.CharacterCreator.DataModel. check why this import is there

Reply:
Second fixed